doc/admin-guide/files/parent.config.en.rst

```diff
@@ -225,6 +225,60 @@ The following list shows the possible actions and their allowed values.
     -  ``consider`` - Use the query string when finding a parent.
     -  ``ignore`` - Do not consider the query string when finding a parent.
 
+.. _parent-config-format-fname:
+
+``fname``
+    One of the following values:
+
+    - ``consider`` - Use the file name string when finding a parent.
+    - ``ignore`` - Do not consider the file name string when finding a parent.
```

Comment:
Default is "consider" same as with the existing qstring option. I followed the same documentation conventions as for the existing options. I will add the a "(default)" tag to both the qstring and fname options although qstring is not mine.